apps/discord-bot/src/commands/bedwars/bedwars.profile.tsx

```diff
@@ -112,3 +113,51 @@ export const BedWarsProfile = ({
     </Container>
   );
 };
+
+export const BedWarsChallenges = ({
+  skin,
+  player,
+  background,
+  logo,
+  tier,
+  badge,
+  t,
+  time,
+}: BaseProfileProps) => {
+  const challenges = player.stats.bedwars.challenges;
+
+  const sidebar: SidebarItem[] = [
+    ["Total Completions", t(challenges.totalChallenges), "§6"],
+    ["Unique Completions", t(challenges.uniqueChallenges), "§e"],
+  ];
+
+  const rows = arrayGroup(Object.entries(challenges).slice(2), 4);
```

Comment:
```suggestion
	delete challenges.totalChallenges;
	delete challenges.uniqueChallenges;
  const rows = arrayGroup(Object.entries(challenges), 4);
```

Objects aren't guaranteed to have the same order.